bodhi/server/models.py

```diff
@@ -2317,6 +2317,8 @@ def new(cls, request, data):
             # For rawhide updates make sure autotime push is enabled
             # https://github.com/fedora-infra/bodhi/issues/3912
             data['autotime'] = True
+            data['stable_days'] = 0
```

Comment:
I don't think that should be hardcoded, it should be something we can configure using the configuration file maybe ? Something like we the mandatory_days_in_testing --> https://pagure.io/fedora-infra/ansible/blob/master/f/roles/bodhi2/base/templates/production.ini.j2#_597

What do you think ?

Reply:
That's about the same question I asked in #4108 : is there any reason for releases not composed by Bodhi to have updates stuck in testing, since no testing repository exists?

The scope of this PR is to prevent users being able to manually set `stable_days` on the update, if we don't want to hardcode there's no need to create another setting, I think we could just override user input with the release default.

Reply:
OK sounds good, I was lacking a bit of context :-) :+1: